src/Elastic.Apm.MongoDb/DiagnosticSource/MongoDiagnosticListener.cs

```diff
@@ -79,10 +79,22 @@ private void HandleCommandStartEvent(CommandStartedEvent @event)
 					{
 						IPEndPoint ipEndPoint => new Destination
 						{
+							Service = new Destination.DestinationService
+							{
+								Resource = $"{ipEndPoint.Address}:{ipEndPoint.Port}"
+							},
 							Address = ipEndPoint.Address.ToString(),
 							Port = ipEndPoint.Port
 						},
-						DnsEndPoint dnsEndPoint => new Destination { Address = dnsEndPoint.Host, Port = dnsEndPoint.Port },
+						DnsEndPoint dnsEndPoint => new Destination
+						{
+							Service = new Destination.DestinationService
```

Comment:
`Service` does not need to be set here. Two things to this:

1. According to the current spec `Service.Resource` should be just `monodb`. Spec [here](https://github.com/elastic/apm/blob/main/specs/agents/tracing-instrumentation-db.md#mongodb)
2. There is a logic in `Span.cs` [here](https://github.com/elastic/apm-agent-dotnet/blob/master/src/Elastic.Apm/Model/Span.cs#L489) which fills this. 

So this should be removed.